compose/cli/main.py

```diff
@@ -633,11 +633,13 @@ def pull(self, options):
         Options:
             --ignore-pull-failures  Pull what it can and ignores images with pull failures.
             --parallel              Pull multiple images in parallel.
+            --quiet                 Pull without printing progress information
```

Comment:
@shin- had forgotten this was still lying around, happy to switch this if it's necessary to merge it